packages/crdt/src/crdt/merkle.ts

```diff
@@ -68,7 +75,7 @@ export function build(timestamps: Timestamp[]) {
   return trie;
 }
 
-export function diff(trie1: TrieNode, trie2: TrieNode): number {
+export function diff(trie1: TrieNode, trie2: TrieNode): number | null {
```

Comment:
This function was missing a return type and also a default return value. Added that